apps/condo/domains/resident/schema/Resident.js

```diff
@@ -101,6 +100,28 @@ const Resident = new GQLListSchema('Resident', {
             access: true,
         },
 
+        organizationFeatures: {
+            schemaDoc: 'Contains features that are enabled for user organization',
+            type: Virtual,
+            extendGraphQLTypes: ['type OrganizationFeatures { hasBillingData: Boolean! }'],
```

Comment:
Should it be moved to separate place to be expandable in future?

Reply:
Please describe what do you mean :) I like the idea, but do not understand what are we trying to accomplish

Reply:
Please refer to `@condo/domains/billing/schema/fields/BillingReceipt`

Reply:
I don't really like this idea. For creating a single field with single type and single fieldset we need to create 2 more files 

We've discussed it in the office -- we are not moving this away, since expandability does not matter in this PR and wouldz
 matter if there is another feature for `organizationFeature`